src/tracing/default.rs

```diff
@@ -2,17 +2,26 @@ use std::collections::HashSet;
 
 use common::ext::OptionExt;
 use serde::{Deserialize, Serialize};
-use tracing_subscriber::prelude::*;
-use tracing_subscriber::{filter, fmt, registry};
+use tracing_subscriber::fmt::{self};
+use tracing_subscriber::{filter, registry, Layer};
 
 use super::*;
 
+#[derive(Default, Clone, Debug, Eq, PartialEq, Deserialize, Serialize)]
+#[serde(rename_all = "lowercase")]
+pub enum LogFormat {
+    #[default]
+    Default,
```

Comment:
Maybe we can name the default `Text`, or something like that. I'm actually not sure what I like best :smiley: 

```suggestion
    #[default]
    Text,
```

Reply:
I like it being ```Text```